src/lib/directive.ts

```diff
@@ -14,15 +14,15 @@
 
 import {Part} from './parts.js';
 
+const directives = new WeakMap<any, Boolean>();
```

Comment:
Too bad IE11 doesn't have WeakSet :/

Reply:
I considered it, but didn't use it for that reason. Also, I don't think using WeakSet vs WeakMap will have an overall performance impact, and using WeakMap may actually be beneficial because it compresses better (as it is used in multiple locations).